docs/user-guide/prototype-an-assistant.rst

```diff
@@ -0,0 +1,198 @@
+The first stories you write should follow the `happy path <glossary>` for your user goal.
+
+A story contains one or more blocks of (user) intent and (bot) `actions <glossary>`_ or `responses <glossary>`_.
+The form you defined above is one kind of action; responses are just bot messages.
+Give intuitive names to your responses starting with ``utter_`` for now; you’ll define what they return later.
+
+Using the general template of the story we have shown you below, write a story or two that serve
+the user goal you have chosen. You can include the steps from your form rule wherever appropriate.
+
+.. code-editor::
+    :language: yaml
+    :id: stories
+    :height: 250
+    :tracking_endpoint: https://trainer-service.prototyping.rasa.com/startPrototyping
+
+    stories:
+     - story: greet and subscribe
+       steps:
+       - intent: greet
+       - action: utter_greet
+       - intent: subscribe
+       - action: newsletter_form
+       - form: newsletter_form
+
+:ref:`Read about best practices for writing stories <stories>`
+
+
+6. Edit responses
+~~~~~~~~~~~~~~~~~
+To give your bot messages to respond to the user with, you need to define responses.
+You can specify one or more text options for each response. If there are multiple, one of the options
+will be chosen at random whenever that response is predicted.
+
+You can add or change text for any of the responses below. If you’re using your own user goal,
+replace the last three responses with the response you used in your stories above.
+
+.. code-editor::
+    :language: yaml
+    :id: responses
+    :height: 300
+    :tracking_endpoint: https://trainer-service.prototyping.rasa.com/startPrototyping
+
+    responses:
+       utter_greet:
+           - text: |
+               Hallo! How can I help you?
+           - text: |
+               hi!
+       utter_ask_email:
+           - text: |
+               What is your email address?
+       utter_subscribed:
+           - text: |
+               I’ve subscribed {email} to the newsletter!
+           - text: |
+               You’ve been subscribed, the newsletter will be sent to {email}.
+
+.. note::
+
+    For this prototype, we have only defined responses, meaning the only thing the assistant does is respond with a
+    predefined message. Custom actions, however, can be defined to do whatever you’d like.
+    For example, for the user goal of subscribing to a newsletter, you could create a custom action
+    that adds the user’s email to a database. You can see an example of this in `Sara's action code <action code>`_.
+
+
+7. Train and run
+~~~~~~~~~~~~~~~~
+
+Rasa has a command line interface that allows you to train and run your bot from a terminal.
+To train your bot on the NLU data, stories and responses you’ve just defined, run ``rasa train`` using
+the button below:
+
+.. train-button::
+    :endpoint: https://trainer-service.prototyping.rasa.com/trainings
+    :method: POST
```

Comment:
This takes a pretty long time -- where is the config stored? can we reduce the epochs?

Reply:
Ah i guess we want to keep it the standard for when it's downloaded though

Reply:
I guess it's fine now